english/light-up/light-up.md

```diff
@@ -1,2 +1,15 @@
-Quick as lightning, you manage to duck the marshmallow hand.
-And with the lighter, you light up one end of the rope and push the other end into the wall.
\ No newline at end of file
+Quick as lightning, you manage to duck the marshmallow hand. And with the lighter, you light up one end of the rope and push the other end into the wall.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.